CMakeLists.txt

```diff
@@ -3,11 +3,18 @@ cmake_minimum_required(VERSION 3.5)
 project(R-Type-Reborn
         LANGUAGES CXX)
 set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 include_directories(includes)
-link_directories(libs)
-link_libraries(StellarForge)
 
-add_executable(R-Type-Reborn ${R-Type-Reborn_SOURCES})
+set(LIBRARY_PATH "${CMAKE_CURRENT_SOURCE_DIR}/libs")
```

Comment:
link_directories does what you want no need of variables

Reply:
This part will be update later when we are going to link the server/client part with our Game engine but for the moment it's not used